<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background:rgb(46, 52, 54)
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="3">
     <widget class="QPushButton" name="btnMult">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(245, 121, 0);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(235, 111, 0);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
      <property name="shortcut">
       <string>*</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="btnMem">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(245, 121, 0);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(235, 111, 0);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>M</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btn2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(239, 41, 41);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(229, 31, 41);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="shortcut">
       <string>2</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btn9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(239, 41, 41);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(229, 31, 41);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
      <property name="shortcut">
       <string>9</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="btnAdd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(245, 121, 0);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(235, 111, 0);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
      <property name="shortcut">
       <string>+</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="btnAc">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(239, 41, 41);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(229, 31, 41);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>Ac</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="btnEqual">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(245, 121, 0);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(235, 111, 0);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
      <property name="shortcut">
       <string>Enter</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btn6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(239, 41, 41);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(229, 31, 41);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <property name="shortcut">
       <string>6</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btnDiv">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(245, 121, 0);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(235, 111, 0);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
      <property name="shortcut">
       <string>/</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QLineEdit" name="display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Courier 10 Pitch</family>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:rgb(11, 11, 11);
color: white;

}</string>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="btnMemDel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(245, 121, 0);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(235, 111, 0);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>M-</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="btnsub">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(245, 121, 0);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(235, 111, 0);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="shortcut">
       <string>-</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(239, 41, 41);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(229, 31, 41);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <property name="shortcut">
       <string>5</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="btnChangeSign">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(239, 41, 41);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(229, 31, 41);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>-/+</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btn1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(239, 41, 41);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(229, 31, 41);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="shortcut">
       <string>1</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btn0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(239, 41, 41);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(229, 31, 41);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="shortcut">
       <string>0</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(239, 41, 41);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(229, 31, 41);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
      <property name="shortcut">
       <string>8</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="btnMemAdd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(245, 121, 0);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(235, 111, 0);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>M+</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="btn3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(239, 41, 41);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(229, 31, 41);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="shortcut">
       <string>3</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(239, 41, 41);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(229, 31, 41);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="shortcut">
       <string>4</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color:rgb(239, 41, 41);
color:rgb(238, 238, 236);
border-radius:1px;
}
QPushButton:pressed{
background-color:rgb(229, 31, 41);
border-radius:1px;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <property name="shortcut">
       <string>7</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
